### PR TITLE
Add warnings for meta descriptions

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/components/arm/xarmlite.md [hunk 1]
```diff
@@ -4,11 +4,12 @@ linkTitle: "xArmLite"
 weight: 34
 type: "docs"
 description: "Configure an xArmLite arm."
+metaDescription: "Configure an xArmLite arm to integrate a UFACTORY Lite 6: Navigate to the Create component menu. Enter a name for your arm, select the type arm, and select the xArmLite model. Click Create component. Edit and fill in the attributes as applicable."
```
Comment (npentrel) [inline docs/components/arm/xarmlite.md:7]:
```suggestion
metaDescription: "Configure a UFACTORY Lite 6 arm: Navigate to the Create component menu. Enter a name for your arm, select the type arm, and select the xArmLite model. Click Create component. Edit and fill in the attributes as applicable."
```

File: docs/components/arm/xarmlite.md [hunk 2]
```diff
@@ -4,11 +4,12 @@ linkTitle: "xArmLite"
...
 tags: ["arm", "components"]
 # SMEs: William Spies
 ---
 
-Configure an `xArmLite` arm to integrate a [UFACTORY Lite 6](https://www.ufactory.cc/product-page/ufactory-lite-6/) into your robot:
+Configure an `xArmLite` arm to integrate a [UFACTORY Lite 6](https://www.ufactory.cc/product-page/ufactory-lite-6/) for your robot:
```
Comment (npentrel) [inline docs/components/arm/xarmlite.md:12]:
```suggestion
Configure an `xArmLite` arm to integrate a [UFACTORY Lite 6](https://www.ufactory.cc/product-page/ufactory-lite-6/) into your robot:
```

File: docs/appendix/troubleshooting.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ linkTitle: "Troubleshooting"
 weight:
 type: "docs"
 description: "A guide to basic troubleshooting of a Viam-based robotic system, with easy fixes to common problems."
+metaDescription: "A guide to basic troubleshooting of a Viam-based robotic system, with easy fixes to common problems.This document lists common errors encountered when working with viam-server and the Viam app, and provides simple steps to resolve them."
```
Comment (npentrel) [inline docs/appendix/troubleshooting.md:7]:
```suggestion
metaDescription: "A guide to basic troubleshooting of a Viam-based robotic system, with easy fixes to common problems. This document lists common errors encountered when working with viam-server and the Viam app, and provides simple steps to resolve them."
```

File: docs/components/arm/xarmlite.md (hunk 2 above)
Comment (andf-viam) [inline docs/components/arm/xarmlite.md:12]:
```suggestion
Configure an `xArmLite` arm to add a [UFACTORY Lite 6](https://www.ufactory.cc/product-page/ufactory-lite-6/) to your robot:
```

File: docs/components/arm/fake.md [hunk 4]
```diff
@@ -4,6 +4,7 @@ linkTitle: "fake"
 weight: 34
 type: "docs"
 description: "Configure a fake arm to use for testing."
+metaDescription: "Configure a fake arm to test different models of robotic arms without any physical hardware: Navigate to the Create component menu. Enter a name for your arm, select the type arm, and select the fake model. Click Create component. Edit and fill in the attributes as applicable."
```
Comment (andf-viam) [inline docs/components/arm/fake.md:7]: If this shows up on the other end in step-wise format (like maybe [this way](https://www.google.com/search?q=how+do+i+delete+cookies+on+ie+6&source=hp&ei=kzJ6ZMerB_GfptQP6tCPuA4&iflsig=AOEireoAAAAAZHpAo41Kh7jXg-vcCihfcKYJGvUdkyMe&ved=0ahUKEwiHwdWxmaX_AhXxj4kEHWroA-cQ4dUDCAs&uact=5&oq=how+do+i+delete+cookies+on+ie+6&gs_lcp=Cgdnd3Mtd2l6EAMyBQghEKABMgUIIRCgATIFCCEQqwIyBQghEKsCMgUIIRCrAlAAWABg0gNoAHAAeACAAX6IAX6SAQMwLjGYAQCgAQKgAQE&sclient=gws-wiz)), this is fine. If this appears instead in paragraph format (like maybe [this way](https://www.google.com/search?q=what+is+a+cookie+in+a+web+browser&ei=wjJ6ZNDuM7jcptQPkvC3WA&ved=0ahUKEwjQ17bImaX_AhU4rokEHRL4DQsQ4dUDCBA&uact=5&oq=what+is+a+cookie+in+a+web+browser&gs_lcp=Cgxnd3Mtd2l6LXNlcnAQAzIECAAQRzIECAAQRzIECAAQRzIECAAQRzIECAAQRzIECAAQRzIECAAQRzIECAAQRzoKCAAQRxDWBBCwA0oECEEYAFDeBVjeBWCgB2gBcAJ4AIABAIgBAJIBAJgBAKABAcABAcgBCA&sclient=gws-wiz-serp)), I'd recommend sticking to sentence case:

```suggestion
metaDescription: "Configure a fake arm to test different models of robotic arms without any physical hardware. Navigate to the create component menu and enter a name for your arm, select the type arm, and select the fake model. Click create component. Edit and fill in the attributes as applicable."
```

Or even:

```suggestion
metaDescription: "Configure a fake arm to test different models of robotic arms without any physical hardware. Navigate to the "Create component" menu and enter a name for your arm, select the type "arm", and select the "fake" model. Click "Create component". Edit and fill in the attributes as applicable."
```

I assume we don't get access to any formatting, right?

Alternatively, consider avoiding including step-based instructions in these descriptions.

File: docs/components/arm/xarmlite.md (hunk 1 above)
Comment (andf-viam) [reply docs/components/arm/xarmlite.md:7]: I agree with this suggestion! Let's merge it in!

File: docs/viam/_index.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
 title: "Viam in 3 minutes"
 linkTitle: "Viam in 3 minutes"
 description: "Viam is a complete software platform that supports every step of your robot development lifecycle."
+metaDescription: "Viam is a complete software platform for robots. It runs on 64-bit Linux and macOS, and provides services for computer vision, motion planning, SLAM, data management, and ML. You can make use of Viam's robot components and extend its capabilities with community-contributed and custom resources."
```
Comment (andf-viam) [inline docs/viam/_index.md:5]:
```suggestion
metaDescription: "Viam is a complete software platform for robots. It runs on 64-bit Linux and macOS, and provides services for computer vision, motion planning, SLAM, data management, and machine learning. You can make use of Viam's pre-defined robot components and extend its capabilities with community-contributed and custom resources."
```

Comment (andf-viam) [reply docs/viam/_index.md:5]: - `pre-defined` - up to you: I think adds clarity to the value of `Viam's robot components` and why a Google Searcher might want to click this link. Or similar "out of the box" word if you've got a better one!

File: layouts/partials/page-description.html [hunk 6]
```diff
@@ -1,5 +1,24 @@
-{{ with .Description | plainify -}}
-  {{ . -}}
+{{ $recentlyChanged := (lt (sub now.Unix .Lastmod.Unix) 3600) }}
```
Comment (andf-viam) [inline layouts/partials/page-description.html:1]: Nice, this is clever!

**Review by andf-viam — APPROVED** [review]
LGTM, some light comments.

Overall though, I somewhat hesitate to put procedural content in these descriptions, as it exposes a lot of issues with step-based content presented in paragraph form. See my comment for a compare. Up to you.

File: layouts/partials/page-description.html (hunk 6 above)
Comment (npentrel) [reply layouts/partials/page-description.html:1]: 😎

File: docs/viam/_index.md [hunk 7]
```diff
@@ -2,6 +2,7 @@
 title: "Viam in 3 minutes"
 linkTitle: "Viam in 3 minutes"
 description: "Viam is a complete software platform that supports every step of your robot development lifecycle."
+metaDescription: "Viam is a complete software platform for robots. It runs on 64-bit Linux and macOS, and provides services for computer vision, motion planning, SLAM, data management, and machine learning. You can make use of Viam's pre-defined robot components and extend its capabilities with community-contributed and custom resources."
```
Comment (npentrel) [inline docs/viam/_index.md:5]:
```suggestion
metaDescription: "Viam is a complete software platform for robots. It runs on 64-bit Linux and macOS, and provides services for computer vision, motion planning, SLAM, data management, and machine learning. You can make use of Viam's pre-defined robot components and community-contributed and custom resources."
```

**Comment by npentrel** [comment]
Not sure about the step based description either - I've accepted your comment for now. We can see how it looks and works out on other pages and adapt

**Pull request merged**